cubric/templates/uwsgi.conf

```diff
@@ -24,3 +24,4 @@ env = PROJECT_HOME={{instancepath}}/{{project}}
 env = DATABASE_URL={{database_url}}
 env = SECRET_KEY='{{secret_key}}'
 env = MEDIA_FOLDER={{instancepath}}/media
+env = {%for e in ENV%}{{e.key}}="{{e.value}}"{% if not loop.last %},{%endif%}{%endfor%}
```

Comment:
Dit is wel heel letterlijk gecopy/paste vanuit supervisor-program.conf en die heeft een andere syntax voor de environment. En inderdaad, database_url wordt dan waarschijnlijk gedupliceerd dus die kan je er uit halen